src/mds/MDLog.cc

```diff
@@ -581,6 +581,27 @@ class C_OFT_Committed : public MDSInternalContext {
   }
 };
 
+void MDLog::try_to_commit_open_file_table()
```

Comment:
This change (commit f623a416da276fa558cc51b4a7d4ddbefdf0def1) is more of a code organization thing, ins't it? The commit message makes it sound otherwise.